### PR TITLE
[FIX] account_move: compute date_last_payment bug during migration

### PR DESCRIPTION
**Pull request by augusto-weiss**

### CODE REVIEW

**Comment by roboadhoc** [comment]
[Pull request status dashboard](https://runbot.adhoc.com.ar/ingadhoc/account-invoicing/pull/130).

File: account_invoice_commission/models/account_move.py [hunk 1]
```diff
@@ -58,11 +58,11 @@ def _compute_partner_user(self):
             users = rec.partner_id.user_ids
             rec.partner_user_id = users and users[0] or False
 
-    @api.depends('line_ids.date')
+    @api.depends('line_ids.amount_residual')
```
Comment (nicomacr) [inline account_invoice_commission/models/account_move.py]:
```suggestion
    @api.depends('payment_group_ids.payment_date')
```

**Review by nicomacr — CHANGES_REQUESTED** [review]
Hi @augusto-weiss ,
Excellent work.
Only one suggest to change.

**Review by nicomacr — DISMISSED** [review]

**Comment by nicomacr** [comment]
@roboadhoc r+

**Comment by roboadhoc** [comment]
@augusto-weiss ya silly goose you can't reopen a PR that's been merged PR.

**Comment by fw-bot-adhoc** [comment]
This pull request has forward-port PRs awaiting action (not merged or closed): ingadhoc/account-invoicing#131

**Comment by fw-bot-adhoc** [comment]
This pull request has forward-port PRs awaiting action (not merged or closed): ingadhoc/account-invoicing#131

**Pull request closed without merging**